tests/drivers/eeprom/testcase.yaml

```diff
@@ -6,3 +6,6 @@ tests:
     build_only: true
     depends_on: eeprom
     tags: drivers userspace
+  peripheral.eeprom.generic.stm32:
+    extra_configs:
+      - CONFIG_EEPROM_STM32=y
```

Comment:
`CONFIG_EEPROM=y` in prj.conf of this test will enable `CONFIG_EEPROM_STM32` automatically (as long as nucleo_l152re.dts enables eeprom node, which is the case).
So, this change is not required.
If something prevents to run tests/compile on nucleo_l152re, there is another issue.

Reply:
added to the list of  a `platform_allow`